gradio/route_utils.py

```diff
@@ -867,13 +867,15 @@ def __init__(self):
         self.ended = False
         self.segment_index = 0
         self.playlist = "#EXTM3U\n#EXT-X-PLAYLIST-TYPE:EVENT\n#EXT-X-TARGETDURATION:10\n#EXT-X-VERSION:4\n#EXT-X-MEDIA-SEQUENCE:0\n"
+        self.max_length = 5
```

Comment:
Why do we hardcode a max length here?

Reply:
It can't be undefined in the http spec but this will be overwritten as soon as we start receiving data

Reply:
Interesting